integrations/rl/puffer/ppo.py

```diff
@@ -145,33 +163,39 @@ def evaluate(data):
             if config.device == "cuda":
                 torch.cuda.synchronize()
 
+        with profile.env:
+            actions = actions.cpu().numpy()
+
+            # Step the environment and reset if done
+            data.vecenv.send(actions)
```

Comment:
Woah, what is this. Why would puffer require this?

Reply:
I’m not sure, this is taken from the puffer code